webclient/bower_components

```diff
@@ -0,0 +1 @@
+../bower_components/
```

Comment:
nope, that happened a while ago when started relying on everything being copied to dist for import to work. It was also compounded recently when the PR that requires browserify to be run got merged (the tests PR). And this doesn't prevent anything, it just doesn't do anything except possibly create circular file structures